rasa/nlu/data_router.py

```diff
@@ -102,7 +103,13 @@ def __init__(self,
         else:
             self.component_builder = ComponentBuilder(use_cache=True)
 
-        self.project_store = self._create_project_store(project_dir)
+        loop = asyncio.get_event_loop()
```

Comment:
I think we should rather do the loop thingy in `rasa/nlu/server.py`, what do you think? Maybe a `initialize_router` method or so which is called after the constructor.

Reply:
I think doing the loop initialization would also make closing the loop safer. Doing this here, might kill the loop for other things.

Reply:
👍 Sounds good. Will change it.